src/Google/Api/Ads/AdWords/Util/v201609/ReportUtilsDelegate.php

```diff
@@ -178,8 +178,9 @@ private static function DownloadReportFromUrl($url, $headers, $params,
       } else if (!empty($error)) {
         $exception = new ReportDownloadException($error);
       } else if (isset($code)) {
-        $exception =
-            new ReportDownloadException('Report download failed.', $code);
+        $message = 'Report download failed. ';
+        !empty($curlError) && $message .= $curlError;
```

Comment:
Could you please use explicit if-clause to be consistent with the style of this library?
```php
if (!empty($curlError)) {
  $message .= $curlError;
}
```